deploy/kubernetes/elastic-agent-kustomize/ksm-autosharding/elastic-agent-standalone/base/elastic-agent-standalone-daemonset.yaml

```diff
@@ -66,6 +66,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            - name: STATE_PATH
```

Comment:
@gizas this isn't a label change, so is this change required?

Reply:
Indeed is not needed

Reply:
It was not me that added this but I will rebase and remove.